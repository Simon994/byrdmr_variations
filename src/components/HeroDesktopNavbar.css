/* Desktop navbar */

.desktop-nav {
  padding-top: 52px;
  display: flex;
  justify-content: space-between;
  position: relative;
}

.left-container ul,
.right-container ul {
  list-style: none;
}

.left-container ul li,
.right-container ul li {
  display: inline-block;
}

.left-container ul {
  margin-left: 131.5px;
}

.right-container ul {
  margin-right: 131.5px;
}

.left-container ul li:first-child,
.right-container ul li:first-child {
  padding-right: 50.5px;
}

