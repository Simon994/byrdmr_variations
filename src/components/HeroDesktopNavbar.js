import MenuAndNavItems from './common/MenuAndNavItems'
import Logo from '../styles/assets/Logo.png'

function HeroDesktopNavbar() {

  return (
    <nav className='desktop-nav'>
      <div className='left-container'>
        <MenuAndNavItems listItems={['store', 'tutorials']}/>
      </div>
      <img 
        src={Logo}
        alt='ByrdMR logo'
        className='byrd-logo'
      />
      <div className='right-container'>
        <MenuAndNavItems listItems={['blog', 'contact']}/>
      </div>
    </nav>
  )
}

export default HeroDesktopNavbar