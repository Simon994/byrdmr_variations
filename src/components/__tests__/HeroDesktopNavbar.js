import { BrowserRouter } from 'react-router-dom'
import renderer from 'react-test-renderer'
import HeroDesktopNavbar from '../HeroDesktopNavbar'

describe('The HeroDesktopNavbar (navbar for hero view on desktop device)', () => {
  it('renders as expected', () => {
    const tree = renderer.create(
      <BrowserRouter>
        <HeroDesktopNavbar/>
      </BrowserRouter>
    ).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
