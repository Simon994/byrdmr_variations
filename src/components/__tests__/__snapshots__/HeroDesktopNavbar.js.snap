// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The HeroDesktopNavbar (navbar for hero view on desktop device) renders as expected 1`] = `
<nav
  className="desktop-nav"
>
  <div
    className="left-container"
  >
    <ul>
      <li>
        <a
          aria-current={null}
          className="nav-link"
          href="/store"
          onClick={[Function]}
        >
          STORE
        </a>
      </li>
      <li>
        <a
          aria-current={null}
          className="nav-link"
          href="/tutorials"
          onClick={[Function]}
        >
          TUTORIALS
        </a>
      </li>
    </ul>
  </div>
  <img
    alt="ByrdMR logo"
    className="sc-bdnxRM iuppKl byrd-logo"
    src="Logo.png"
  />
  <div
    className="right-container"
  >
    <ul>
      <li>
        <a
          aria-current={null}
          className="nav-link"
          href="/blog"
          onClick={[Function]}
        >
          BLOG
        </a>
      </li>
      <li>
        <a
          aria-current={null}
          className="nav-link"
          href="/contact"
          onClick={[Function]}
        >
          CONTACT
        </a>
      </li>
    </ul>
  </div>
</nav>
`;
